config/data_centers/data_centers.json

```diff
@@ -679,6 +679,36 @@
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": "2021"
   },
+  "hlrs-de-stuttgart": {
+    "provider": "hlrs",
+    "displayName": "HLRS Stuttgart",
+    "zoneKey": "DE-BW",
+    "lonlat": [9.0970622, 48.739457],
+    "region": "de-stuttgart",
+    "source": "https://www.hlrs.de/",
+    "operationalSince": null,
+    "status": "operational"
+  },
+  "data-center-stuttgart-vaihingen-1": {
+    "provider": "itenos",
+    "displayName": "Itenos data center Stuttgart-Vaihingen-1",
+    "zoneKey": "DE-BW",
+    "lonlat": [9.1285794, 48.7214015],
+    "region": "de-stuttgart",
+    "source": "https://itenos.de/en/data-center/data-center-germany/data-center-stuttgart-1/",
+    "operationalSince": null,
+    "status": "operational"
+  },
+  "jsc-de-juelich": {
+    "provider": "jsc",
+    "displayName": "Jülich Supercomputing Centre",
+    "zoneKey": "DE-NW",
+    "lonlat": [6.3996038, 50.9022892],
+    "region": "de-juelich",
+    "source": "https://www.fz-juelich.de/ias/jsc/EN/Home/home_node.html",
+    "operationalSince": null,
+    "status": "operational"
+  },
```

Comment:
Is this a datacenter or data lab?
I think we just want to stick with datacenters for now.
@jbdietrich any opinion?

Reply:
I am not sure, I found this quote on the source site -> JUPITER at the Forschungszentrum Jülich in North Rhine-Westphalia, Germany, becomes the fastest supercomputer in Europe. JUPITER, developed by the Jülich Supercomputing Centre (JSC) and owned by EuroHPC JU, ranks 4th on the [June 2025 TOP500 ](https://top500.org/lists/top500/2025/06/)list of the world’s fastest supercomputers